src/components/Link.tsx

```diff
@@ -48,7 +48,6 @@ const Link = React.memo(({ simplePath, label, charLimit = 32, style, cssRaw, cla
         className,
       )}
       {...fastClick(e => {
-        // eslint-disable-line react/no-danger-with-children
```

Comment:
?

Reply:
Basically as per eslint v9.

if there is no rule define in eslint.config.ts and we are adding the command in our code. so eslint lint will show that command as warning.

so we just fix all these by running yarn lint --fix. if you want to keep these warning as it is so will revert all these.